tap/tapify.py

```diff
@@ -0,0 +1,84 @@
+"""Tapify module, which can initialize a class or run a function by parsing arguments from the command line."""
+from inspect import signature, Parameter
+from typing import Any, Callable, List, Optional, TypeVar, Union
+
+from docstring_parser import parse
+
+from tap import Tap
+
+InputType = TypeVar('InputType')
+OutputType = TypeVar('OutputType')
+
+
+def tapify(class_or_function: Union[Callable[[InputType], OutputType], OutputType],
+           args: Optional[List[str]] = None,
+           known_only: bool = False,
+           **func_kwargs) -> OutputType:
+    """Tapify initializes a class or runs a function by parsing arguments from the command line.
+
+    :param class_or_function: The class or function to run with the provided arguments.
+    :param args: Arguments to parse. If None, arguments are parsed from the command line.
+    :param known_only: If true, ignores extra arguments and only parses known arguments.
+    :param func_kwargs: Additional keyword arguments for the function. These act as default values when
+                        parsing the command line arguments and overwrite the function defaults but
+                        are overwritten by the parsed command line arguments.
+    """
+    # Get signature from class or function
+    sig = signature(class_or_function)
+
+    # Parse class or function docstring in one line
+    if isinstance(class_or_function, type) and class_or_function.__init__.__doc__ is not None:
+        doc = class_or_function.__init__.__doc__
+    else:
+        doc = class_or_function.__doc__
+
+    # Parse docstring
+    docstring = parse(doc)
+
+    # Get the description of each argument in the class init or function
+    param_to_description = {param.arg_name: param.description for param in docstring.params}
+
+    # Create a Tap object
+    tap = Tap(description=f'{docstring.short_description}\n{docstring.long_description}')
```

Comment:
Perhaps:

```suggestion
    tap = Tap(description="\n".join(filter(None, (docstring.short_description, docstring.long_description))))
```

in case `.long_description` is `None`?

Reply:
Great find. Thank you!